packages/cashier/src/pages/withdrawal/withdrawal-crypto-receipt/withdrawal-crypto-receipt.tsx

```diff
@@ -1,41 +1,68 @@
 import React from 'react';
 import { Button, Clipboard, Icon, Text } from '@deriv/components';
-import { isMobile } from '@deriv/shared';
+import { useCryptoTransactions } from '@deriv/hooks';
+import { TModifiedTransaction } from '@deriv/hooks/src/useCryptoTransactions';
 import { observer, useStore } from '@deriv/stores';
 import { Localize, localize } from '@deriv/translations';
-import { useCashierStore } from '../../../stores/useCashierStores';
-import { TAccount } from '../../../types';
-import { getAccountText } from '../../../utils/utility';
+import { useCashierStore } from 'Stores/useCashierStores';
+import { TAccount } from 'Types';
+import { getAccountText } from 'Utils/utility';
+import { cryptoTransactionMapper } from '../../../modules/transactions-crypto/helpers';
 import './withdrawal-crypto-receipt.scss';
 
 type TWalletInformationProps = {
     account: TAccount;
     blockchain_address: string;
+    is_mobile: boolean;
 };
 
-const Status = () => {
+type TStatusProps = {
+    last_transaction:
+        | (TModifiedTransaction & {
+              is_deposit: boolean;
+              is_withdrawal: boolean;
+          })
+        | undefined;
+};
+
+const Status = ({ last_transaction }: TStatusProps) => {
+    const [status, setStatus] = React.useState({ status_color: 'warning', status_name: 'In review' });
```

Comment:
why do we need a state for this? can `cryptoTransactionMapper` not handle this?

Reply:
I wanted to have some default values for the statuses since last_transaction can be undefined, and I didn't want to call cryptoTransactionMapper outside of the useEffect since it would give me an error that last_transaction is undefined. So its a combination to check for last_transaction updates and then have the useEffect handle what the status is